staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go

```diff
@@ -98,6 +98,26 @@ func (c *APIServiceRegistrationController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	/// initially sync all APIServices to make sure the proxy handler is complete
+	if err := wait.PollImmediateUntil(time.Second, func() (bool, error) {
+		services, err := c.apiServiceLister.List(labels.Everything())
+		if err != nil {
+			utilruntime.HandleError(fmt.Errorf("failed to initially list APIServices"))
+			return false, nil
+		}
+		for _, s := range services {
+			if err := c.apiHandlerManager.AddAPIService(s); err != nil {
+				utilruntime.HandleError(fmt.Errorf("failed to initially sync APIService %s", s.Name))
```

Comment:
here too, why not to pass the err as well?

Reply:
good point

Reply:
fixed